libs/net/models/Areas/Admin/Report/FilterModel.cs

```diff
@@ -23,12 +23,12 @@ public class FilterModel : BaseTypeWithAuditColumnsModel<int>
     /// <summary>
     /// get/set - The Elasticsearch query.
     /// </summary>
-    public Dictionary<string, object> Query { get; set; } = new Dictionary<string, object>();
+    public JsonDocument Query { get; set; } = JsonDocument.Parse("{}");
```

Comment:
I've been moving away from `Dictionary` in some situations because we don't need to access the values.